lib/msal-browser/docs/initialization.md

```diff
@@ -62,6 +62,12 @@ const msalConfig = {
 };
 ```
 
+**Note**: The `protocolMode` configuration option changes the following behavior:
```

Comment:
Nit: Can we add more context on why we added this? Any spec we can link to?

Reply:
No specs, no. Can add more context as to the purpose.